src/objects/Error__c.object

```diff
@@ -144,6 +144,29 @@
         <trackTrending>false</trackTrending>
         <type>LongTextArea</type>
         <visibleLines>10</visibleLines>
+    </fields>
+     <fields>
+        <fullName>Record_Id__c</fullName>
+        <description>The Salesforce Id of the record associated with this error.</description>
+        <externalId>true</externalId>
+        <inlineHelpText>The Salesforce Id of the record associated with this error.</inlineHelpText>
+        <label>Record Id</label>
+        <length>255</length>
```

Comment:
Is there a need for this field to be 255 instead of 18? I guess we can't configure a field as indexed unless it's unique and I can see why you wouldn't want this as unique. I recommend updating the Gem docs to include a note about indexing this field or the `Retry_Pending__c` field for performance **if** the batch job starts to time out.

Reply:
No...That was a screw up.  I will get it changed.  It is (supposed) to be Text(18) External Id (Non unique) which will index it.

The index should be across Category_Type__c, Record_ID__c, and Retry_Pending__c, I think.